src/renderers/ListRenderer.php

```diff
@@ -364,4 +384,27 @@ protected function prepareTemplate()
     {
         return $this->renderRowContent();
     }
+
+    /**
+     * Returns an array of JQuery sortable plugin options for ListRenderer
+     * @return array
+     */
+    protected function getJsSortableOptions()
+    {
+        return [
+            'containerSelector' => '.list-renderer',
```

Comment:
i think it is better to call parent method and just override `containerSelector`

Reply:
For table
```php
            'containerSelector' => 'table',
            'itemPath'          => '> tbody',
            'itemSelector'      => 'tr',
```
for list
```php
            'containerSelector' => '.list-renderer',
            'itemSelector' => '.multiple-input-list__item',
```

Reply:
So... I try with
```php
    protected function getJsSortableOptions()
    {
        return ArrayHelper::merge(parent::getJsSortableOptions(),
        [
            'containerSelector' => '.list-renderer',
            'itemPath' => new UnsetArrayValue,
            'itemSelector' => '.multiple-input-list__item',
        ]);
    }
```

Reply:
look like work fine